packages/cache-utils/ava.config.js

```diff
@@ -0,0 +1,8 @@
+import baseConfig from '../../ava.config.js'
```

Comment:
```suggestion
import baseConfig from '../../ava.base.config.js'
```

I would rename the root config to a base config